fetch.bs

```diff
@@ -8713,83 +8713,140 @@ relevant to you. The rest of the parameters are used less frequently, often for
+<h3 id=fetch-elsewhere-fetch class=no-num>Processing Responses</h3>
 
-<p>Aside from a <a for=/>request</a> the <a for=/>fetch</a> operation takes several optional
-arguments. For those arguments that take an algorithm: the algorithm will be called from a task (or
-in a <a for=/>parallel queue</a> if <a for=fetch><i>useParallelQueue</i></a> is true).
+<p>Once the <a for=/>request</a> is set up, determine how you would like to process the
+<a for=/>response</a>, and in particular at what stage you would like to receive a callback:
 
 <dl>
- <dt><a for=fetch><i>processRequestBodyChunkLength</i></a>
- <dd><p>Takes an algorithm that will be passed the number of bytes that have been transmitted from
- the <a for=/>request</a>'s <a for=request>body</a>. The algorithm will be invoked for each
- transmitted chunk. Most standards will not need this.
-
- <dt><a for=fetch><i>processRequestEndOfBody</i></a>
- <dd><p>Takes an algorithm that will be passed nothing. Indicates <a for=/>request</a>'s
- <a for=request>body</a> has been transmitted. Most standards will not need this.
-
- <dt><a for=fetch><i>processEarlyHintsResponse</i></a>
- <dd><p>Takes an algorithm that will be passed a <a for=/>response</a> (whose
- <a for=response>status</a> is 103). Can only be used for navigations as defined by
- <cite>HTML</cite>. [[HTML]]
-
- <dt><a for=fetch><i>processResponse</i></a>
+ <dt>Upon Completion
  <dd>
-  <p>Takes an algorithm that will be passed a <a for=/>response</a>. Indicates
-  <a for=/>response</a>'s <a for=response>header list</a> has been received and initialized. This
-  is primarily useful for standards that want to operate on <a for=/>response</a>'s
-  <a for=response>body</a>'s <a for=body>stream</a> directly.
-
-  <p>If the <a for=/>request</a>'s <a for=request>mode</a> is "<code>navigate</code>" and its
-  <a for=request>redirect mode</a> is "<code>manual</code>", then callers need to follow a very
-  specific flow with this algorithm to get the intended behavior. They should compute the
-  appropriate <a for=response>location URL</a>, and if it is non-null or failure, then they should
-  call <a for="fetch controller">process the next manual redirect</a>. This will result in
-  <a for=fetch><i>processResponse</i></a> being called again, with the next <a for=/>response</a>
-  in the redirect chain.
-
- <dt><a for=fetch><i>processResponseEndOfBody</i></a>
- <dd><p>Takes an algorithm that will be passed a <a for=/>response</a>. Indicates the network is
- done transmitting the response. This does not read <a for=/>response</a>'s
- <a for=response>body</a>.
+  <p>This is the most common way in which clients handle a <a for=/>response</a>, for example
+  <a lt="fetch a classic script">scripts</a> and <a lt="fetch a style resource">style resources</a>.
+  The <a for=/>response</a>'s <a for=response>body</a> is downloaded in its entirety into a
+  <a>byte sequence</a>, and then processed by the <a for=request>client</a>.
```

Comment:
```suggestion
  <a>byte sequence</a>, and then processed by the client.
```

Reply:
I don't think this suggestion has been adopted yet, has it? I was going to post the same comment as Jeffrey did last week it seems but I figured I'd look through the comments to see if it had already been made. But the sentence appears to still be here: https://whatpr.org/fetch/1614.html#fetch-elsewhere-fetch:~:text=sequence%2C%20and%20then%20processed%20by%20the%20client.

Reply:
Or... maybe the PR preview is out of date with the spec text? I'm not sure what's happening...